### ISSUE TITLE
TableCaption Node's start/end indices are not set

### ISSUE THREAD

**Issue by vsch** [sirthias/pegdown#175]
The TableCaption node is not properly created in the parser resulting in startIndex, endIndex always being zero.

I have a fix and an AST test, here is the result of the new AST test without the fix, it shows the problem well:

```
[error]          TableCaptionNode [0-0]
[error]            TextNode [604-609] 'Table'
[error]            SpecialTextNode [609-610] ':'
[error]            TextNode [610-618] ' Caption'
```

Will post PR with a bunch of other parser compatibility and other source range anomalies.

**Comment by sirthias** [comment]
You rock! :)

**Comment by sirthias** [comment]
Closed by #181.

**Comment by vsch** [comment]
Thank the plugin. Visually highlighting the range makes finding bugs a piece of cake. I now have a “Debug” scheme in the plugin for myself. It has neon background colours for all the elements so that you can easily see where they start and end.

Best regards,


Vladimir.
